docs/usage/install/underlay/get-started-host-device-zh_CN.md

```diff
@@ -0,0 +1,159 @@
+# Host-device-cni Quick Start
+
+[**English**](./get-started-host-device.md) | **简体中文**
+
+Spiderpool 可用作 Underlay 网络场景下提供固定 IP 的一种解决方案，本文将以 [Multus](https://github.com/k8snetworkplumbingwg/multus-cni)、[Host-Device](https://github.com/containernetworking/plugins/tree/main/plugins/main/host-device) 、[Spiderpool](https://github.com/spidernet-io/spiderpool) 为例，搭建一套完整的 Underlay 网络解决方案，该方案能将可用的网桥公开为节点资源，供集群使用。
+
+[`host-device`](https://github.com/containernetworking/plugins/tree/main/plugins/main/host-device)  是一个用于把指定的设备从宿主机的网络命名空间移到容器的网络命名空间下CNI插件。
+
+## 先决条件
+
+1. [安装要求](./../system-requirements-zh_CN.md)
+
+2. 一个多节点的 Kubernetes 集群
+
+3. [Helm 工具](https://helm.sh/docs/intro/install/)
+
+4. 如果您使用如 Fedora、Centos 等 OS， 并且使用 NetworkManager 管理和配置网络，在以下场景时建议您需要配置 NetworkManager:
+
+    * 如果你使用 Underlay 模式，`coordinator` 会在主机上创建 veth 接口，为了防止 NetworkManager 干扰 veth 接口, 导致 Pod 访问异常。我们需要配置 NetworkManager，使其不纳管这些 Veth 接口。
+
+    * 如果你通过 `Iface`r 创建 Vlan 和 Bond 接口，NetworkManager 可能会干扰这些接口，导致 Pod 访问异常。我们需要配置 NetworkManager，使其不纳管这些 Veth 接口。
+
+      ```shell
+      ~# IFACER_INTERFACE="<NAME>"
+      ~# cat > /etc/NetworkManager/conf.d/spidernet.conf <<EOF
+      [keyfile]
+      unmanaged-devices=interface-name:^veth*;interface-name:${IFACER_INTERFACE}
+      EOF
+      ~# systemctl restart NetworkManager
+      ```
+
+## 安装 Spiderpool
+
+1. 安装 Spiderpool。
+
+    ```bash
+    helm repo add spiderpool https://spidernet-io.github.io/spiderpool
+    helm repo update spiderpool
+    helm install spiderpool spiderpool/spiderpool --namespace kube-system --set multus.multusCNI.defaultCniCRName="host-device-conf" --set plugins.installHostDeviceCNI=true
```

Comment:
remove `--set plugins.installHostDeviceCNI=true` because `--install-cni` would install the whole cni-plugins including `host-device`.

Reply:
https://github.com/spidernet-io/spiderpool/blob/main/images/spiderpool-plugins/entrypoint.sh